arwen/host/execution.go

```diff
@@ -930,7 +898,7 @@ func (host *vmHost) callSCMethod() error {
 		err = host.handleBreakpointIfAny(err)
 	}
 	if err == nil {
-		err = host.checkFinalGasAfterExit()
+		// err = host.checkFinalGasAfterExit()
```

Comment:
this is a must :D

Reply:
Reverted. Accidentally forgot to uncomment it...